src/bioregistry/data/bioregistry.json

```diff
@@ -2885,18 +2890,18 @@
     "miriam": {
       "deprecated": false,
       "description": "An Archival Resource Key (ARK) is a Uniform Resource Locator (URL) that is a multi-purpose persistent identifier for information objects of any type.",
-      "homepage": "http://n2t.net/",
+      "homepage": "https://arks.org/",
       "id": "00000592",
       "name": "ARK",
       "namespaceEmbeddedInLui": true,
       "pattern": "^(ark\\:)/*[0-9A-Za-z]+(?:/[\\w/.=*+@\\$-]*)?(?:\\?.*)?$",
       "prefix": "ark",
-      "sampleId": "/12345/fk1234",
+      "sampleId": "/53355/cl010066723",
```

Comment:
Since this is from MIRIAM, it will get overrided. Instead, we can make an override with a top-level `example` key containing your update.

Reply:
See c64101c6